### PR TITLE
Add commands to manage passwords for a database branch

### PR DESCRIPTION
**Pull request by Phanatic**
This PR adds a few commands to manage passwords for a given database branch in PlanetScale.

Operations included are : 
- [x] Create a new password on a database branch.
- [x] List all  password on a database branch.
- [x] Delete a  password on a database branch.

Depends on this PR : 
* https://github.com/planetscale/planetscale-go/pull/86

### CODE REVIEW

File: internal/cmd/passwords/passwords.go [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+func toPasswordWithPlainText(password *ps.DatabaseBranchPassword) *PasswordWithPlainText {
+	return &PasswordWithPlainText{
+		Name:              password.Name,
+		PublicID:          password.PublicID,
+		PlainText:         password.PlainText,
+		AccessHostUrl:     password.Branch.AccessHostURL,
+		Role:              password.Role,
+		RoleDesc:          toRoleDesc(password.Role),
+		orig:              password,
```
Comment (fatih) [inline internal/cmd/passwords/passwords.go:108]: nit: can you move this to the bottom. We should make sure it doesn't get removed in the future accidentally.

File: internal/cmd/passwords/passwords.go [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+		PersistentPreRunE: cmdutil.CheckAuthentication(ch.Config),
+	}
+
+	cmd.PersistentFlags().StringVar(&ch.Config.Organization, "org", ch.Config.Organization, "The organization for the current user")
+	cmd.MarkPersistentFlagRequired("org") // nolint:errcheck
+
+	cmd.AddCommand(CreateCmd(ch))
+	cmd.AddCommand(ListCmd(ch))
+	cmd.AddCommand(DeleteCmd(ch))
+	// cmd.AddCommand(ShowCmd(ch))
```
Comment (fatih) [inline internal/cmd/passwords/passwords.go:29]: Do you plan to add this command or should we remove this line? The description also says:

```
Create, list, show, and delete branch passwords
```

**Review by fatih — APPROVED** [review]
Looks good to me 👍🏼

File: internal/cmd/passwords/passwords.go (hunk 2 above)
Comment (Phanatic) [reply internal/cmd/passwords/passwords.go:29]: I'll remove it.

**Pull request merged**